lib/active_record/virtual_attributes.rb

```diff
@@ -91,18 +94,12 @@ def load_schema!
           type = type.call if type.respond_to?(:call)
           type = ActiveRecord::Type.lookup(type, **options.except(:uses, :arel)) if type.kind_of?(Symbol)
 
-          define_virtual_attribute(name, type, **options.slice(:uses, :arel))
-        end
-
-        virtual_delegates_to_define.each do |method_name, (method, options)|
-          define_virtual_delegate(method_name, method, options)
+          define_virtual_attribute(name, type)
```

Comment:
Nice, this is much clearer with the uses and arel handled elsewhere.  This method has a single responsibility now.

Reply:
I was debating dropping the `options` here, since it is only used very deep in the `Type.lookup` discovery code, and as far as I know, we don't use this. But maybe array or something leverages this, so punting and leaving it in